### DISCUSSION TITLE
Filament Design System

### DISCUSSION THREAD

**Discussion by DiazFarindra** [filamentphp/filament#11656] [General]
hello teams, I just want to know if filament has documentation related to the design system that you use to create the filament interface system?

I've been surfing the internet but didn't find anything, someone made it for filament v2 but it turned out to be not official and incomplete, maybe one day your team wants to release documentation about the design system you use with the Figma platform or something like that.

thanks in advance

**Comment by triasrahman** [comment]
I found a pretty good Figma template that might help: 

https://www.figma.com/community/file/1417716904167561805/filament-v3-free

It’s not official, but it’s definitely worth checking out if you’re looking for some design resources for Filament.

Hope it helps!
